tests/vm_test.go

```diff
@@ -582,14 +582,10 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 			By("Ensuring the VirtualMachineInstance is removed")
 			Eventually(ThisVMIWith(vm.Namespace, vm.Name), 240*time.Second, 1*time.Second).ShouldNot(Exist())
-
-			vm, err = virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
-			Expect(err).ToNot(HaveOccurred())
+			By("Ensuring stateChangeRequests list gets cleared")
+			Eventually(ThisVM(vm), 60*time.Second, 1*time.Second).Should(Not(HaveStateChangeRequests()))
```

Comment:
The below, assert would fail because we do not fetch the vm again. Also the timeout should be lower.

Reply:
Does 30 seconds works ?